src/pages/Farm/reward.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react'
+import AppBody from '../AppBody'
+import styled from 'styled-components'
+import { RouteComponentProps } from 'react-router-dom'
+import Apy from '../../components/RewardCards/info'
+import vector from '../../assets/svg/vector.svg'
+import deposits from '../../assets/svg/deposits.svg'
+import rewards from '../../assets/svg/rewardsAcc.svg'
+import Icon from '../../components/FarmTable/icons'
+import apyPurple from '../../assets/svg/questionmark.svg'
+import apyBlue from '../../assets/svg/questionmark2.svg'
+import apyGreen from '../../assets/svg/questionmark3.svg'
+import Reselect from '../../components/RewardCards'
+import { useActiveWeb3React } from '../../hooks'
+import Config from '../../constants/abis/config.json'
+import { fetchStakerInfo, fetchStats } from '../../utils/farm'
+import { getLPBalance } from '../../utils/rewards'
+import { Reward, RewardObj, User, UserObj } from '../../utils/farm/constants'
+
+const Container = styled('div')`
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  padding-left: 24%;
+  padding-right: 24%;
+  margin-bottom: 50px;
+  text-align: left;
+  z-index: 3;
+  > span {
+    font-size: 32px;
+    font-weight: 600;
+    line-height: 38px;
+  }
+`
+
+const Wrapper = styled('div')`
+  width: 100%;
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  flex-wrap: wrap;
+  > span {
+    line-height: 54px;
+    font-size: 32px;
+    font-weight: 500;
+  }
+`
+const Item = styled('div')`
+  padding: 4px;
+  width: 33%;
+  @media only screen and (max-width: 768px) {
+    width: 100%;
+  }
+`
+
+export default function FarmReselect(props: RouteComponentProps<{ address: string }>) {
+  const {
+    match: {
+      params: { address }
+    }
+  } = props
+  const obj: { [index: string]: any } = Config[0].contracts.fuse
+  const { account, library } = useActiveWeb3React()
+  const [result, setResult] = useState<Reward>(RewardObj)
+  const [user, setUser] = useState<User>(UserObj)
+
+  useEffect(() => {
+    getLPBalance(obj[address].LPToken, account).then(res => {
+      setUser({ ...user, lpAvailable: res })
+    })
+
+    fetchStakerInfo(address, library?.provider, 'multi', account ? account : '').then(res =>
+      setUser({
+        ...user,
+        lpDeposited: (res[0] / 1000000000000000000).toFixed(2)
```

Comment:
there are internal functions for parsing and formatting numbers

Reply:
Can you please point me directly to the lib we have and function necessary? Tried formatWeth from ethers but no avail :/